examples/data-block-examples/landsat8-geotiff-example.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "straight-candy",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/wilhelmina.nekoto/.pyenv/versions/UP42-py-dev/lib/python3.7/site-packages/pandas/compat/__init__.py:97: UserWarning: Could not import the lzma module. Your installed Python is incomplete. Attempting to use lzma compression will result in a RuntimeError.\n",
```

Comment:
Please either solve this problem or remove the output of the cell from the jupyter notebook if you think it is not important to resolve. In any case, should not be visible to users.